ddtrace/debugging/_async.py

```diff
@@ -2,7 +2,7 @@
 from types import CoroutineType
 from typing import List
 
-from ddtrace.debugging._snapshot.collector import SnapshotContext
+from ddtrace.debugging._capture.collector import SnapshotContext
```

Comment:
Does this renaming reflect the new data model?

Reply:
new data model and also simplification of collector.